Datasets

matthewhirschey opened the pull request:
Major package update. Key changes include:

- update all data object function arguments with consistent naming
- name matches object created in ddh_data
- updates download and load functions

dleehr commented:
@hirscheylab I've run the Makefile on HARDAC and uploaded the data to DDS. I'll next be loading the data onto the web server and sharing the files with you. The Makefile should work on your machine as well as the cluster, but let me know if you have questions.

matthewhirschey committed:
fix umap signature/sequence naming

matthewhirschey committed:
fix umap signature/sequence naming

matthewhirschey committed:
re-write download and load funs

Simplifies downloading by assuming everything is in a single bucket in a dir called _data. if test = TRUE, then grabs those data and returns from teh function. if private = TRUE, then grabs data from that bucket, and continues. Grabs data from usual bucket. Also, loading function now loads data from dir, as before, but will drop the suffix _test of the object name (not the file name). Now loads only feather files.

matthewhirschey committed:
adds ability to load object "groups"

Three distinct possibilities for loading objects. First, all objects (default). Second, you can load an object group, by indicating a "type". Third, you can load you can load a single objects

matthewhirschey committed:
replace EVERY data object

data objects as arguments in functions now have a consistent naming, and each should also have a default value matching the name of an object that is generated in ddh_data

matthewhirschey committed:
replace EVERY data object

data objects as arguments in functions now have a consistent naming, and each should also have a default value matching the name of an object that is generated in ddh_data

pcastellanoescuder committed:
bump version, update documentation, and fix one minor bug in documentation

pcastellanoescuder committed:
bump version, remove load_rds, and clean dependencies

pcastellanoescuder committed:
test

pcastellanoescuder reviewed (approved):
LGTM.

matthewhirschey committed:
fix load_feather_object

need to assign to global environment (not fun environment)

pcastellanoescuder committed:
naming in graphs 5

pcastellanoescuder committed:
naming in graphs 51

pcastellanoescuder committed:
bug in plots

pcastellanoescuder committed:
bug in helpers

pcastellanoescuder committed:
bug in links

pcastellanoescuder committed:
test

pcastellanoescuder committed:
testing DONE

pcastellanoescuder committed:
fix conflicts
